deepinpy/utils/utils.py

```diff
@@ -208,3 +209,88 @@ def ifft2(x, axes=(-2, -1)):
     """
 
     return scipy.fftpack.ifft2(x, axes=axes)
+
+def _tf_fspecial_gauss(size: int, sigma: float) -> torch.Tensor:
+    """Function to mimic the 'fspecial' gaussian MATLAB function. Credit to Ke Wang (kewang@berkeley.edu)
+        Used to compute SSIM loss.
+
+    Args:
+        size (int): Shape of Gaussian filter
+        sigma (float): Standard deviation of the Gaussian filter
+
+    Returns:
+        (Tensor) Gaussian filter
+    """
+    x_data, y_data = np.mgrid[
+        -size // 2 + 1 : size // 2 + 1, -size // 2 + 1 : size // 2 + 1
+    ]
+
+    x_data = np.expand_dims(x_data, axis=0).astype(np.float32)
+    x_data = np.expand_dims(x_data, axis=0).astype(np.float32)
+
+    y_data = np.expand_dims(y_data, axis=0)
+    y_data = np.expand_dims(y_data, axis=0)
+
+    g = torch.exp(
+        -(
+            (torch.from_numpy(x_data) ** 2 + torch.from_numpy(y_data) ** 2)
+            / (2.0 * torch.tensor(sigma) ** 2)
+        )
+    )
+    return g / torch.sum(g)
+
+def ssim_loss(y_pred, y_true):
```

Comment:
Is there a reason you are using this over the SSIM loss function defined using `skimage` (https://github.com/facebookresearch/fastMRI/blob/main/fastmri/evaluate.py#L40) ?

Reply:
This one natively uses torch tensors and we can also change the weighting of alpha, beta, and gamma in the SSIM calculation. (it's also from GE research, so we figured it's pretty reputable)

Reply:
sounds good!